objectDataTypes.md

```diff
@@ -0,0 +1,149 @@
+
+# Object Data Types
+## Practice Worksheet
+
+
+### Ruby Operations on Basic Object Types
+
+Without putting the code into Ruby or IRB predict the output and data-type of the following?
```

Comment:
No ? mark

It might also be good to just say that once they complete the worksheet, they _should_ run it in irb to verify their answers.